manim/mobject/types/image_mobject.py

```diff
@@ -171,15 +178,15 @@ def construct(self):
 
     def __init__(
         self,
-        filename_or_array,
+        filename_or_array: str | pathlib.PurePath | npt.NDArray,
```

Comment:
Maybe use `manim.typing.StrPath`?
```suggestion
        filename_or_array: StrPath | npt.NDArray,
```

Reply:
Much better approach.